### DISCUSSION TITLE
Select "Rooms" as default

### DISCUSSION THREAD

**Discussion by luk6173** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#388] [Q&A] [answered]
Hi,

is it possible to select "Rooms" as default selection? Every Time I open my Tab "Path" is default selected.

![grafik](https://user-images.githubusercontent.com/24668586/164643299-41bdcb62-c0a5-4f87-b47e-021ecec4be0e.png)

Thank you and Regards.

**Comment by PiotrMachowski** [comment, marked as answer]
You just have to change order of entries in `map_modes` section
